src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public Rectangle(String color, int bothside, int height) {
+        super(color);
+        this.bothside = bothside;
+        this.height = height;
+
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Rectangle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Both side length: " + bothside);
+        System.out.println("Height length: " + height);
+        System.out.println("Area: " + getArea());
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (bothside * height) / 2;
```

Comment:
The formula for the area of a rectangle is incorrect. It should be 'length * width' without dividing by 2, which is the formula for the area of a triangle.